### ISSUE TITLE
Is there a way to get all the posts in a public page?

### ISSUE THREAD

**Issue by lshowo** [kevinzg/facebook-scraper#610]
I don't know how to get the exact page count of a public page, so can I assign a very large number to the page parameter to get all the posts?

**Comment by neon-ninja** [comment]
yes, but it might take a very long time. You can also set it to `None` to disable this limit
